### DISCUSSION TITLE
Received Someone Else's GitHub Graduation 2022 Developer Card

### DISCUSSION THREAD

**Discussion by tnguyen372** [community/community#18501] [GitHub Education] [answered]
I received someone else's github developer card instead of my own that were given out for the [GitHub Graduation 2022 event](https://github.com/education/GitHubGraduation-2022). We both have the same first name so I guess it was an honest mix up when they shipped out the cards. The GitHub Graduation 2022 repo doesn't have any information on who to contact for issues and there weren't any discussion posts that mentioned this same issue so I made this post.

I'm not sure who to contact at GitHub to resolve this issue right now so I'd appreciate any suggestions! My only course of action for now is to just contact the person whose card I received and find a way to ship our cards to each other.

**EDIT:**
The GitHub Graduation team is working on reprinting cards and mailing them to graduates who received the wrong card in the U.S. 
Credit to LambdaPaul for providing the [source](https://github.com/education/GitHubGraduation-2022/issues/8312).
Unfortunately, I cannot mark LambdaPaul's reply as the answer because it is not a direct reply to my question.

**Comment by baysideengineer** [comment, marked as answer]
Hello @tnguyen372, sounds like a classic mix-up. I think best would be to [contact](https://support.github.com/contact/education?tags=hubberfy_education) GitHub directly. I am sure they can handle this for you. Congratulations for graduating!

**Reply by tnguyen372** [reply]
Thanks! I'll send a request to GitHub Support. Hopefully, there weren't other graduates who ran into the same issue.

**Comment by pfei-sa** [comment]
Also received someone's card (@lambdapaul I've got your card 😭).

**Reply by paulwxyz** [reply]
I did too (@PaulBGD, I got yours). So did a lot of people in the US (https://github.com/education/GitHubGraduation-2022/issues/8312)

It seems like the addressee info and GitHub account names are off by 1 (or a small number) since our names are so close alphabetically.

**Reply by PaulBGD** [reply]
@LambdaPaul welp, I hope you enjoy it!
